### ISSUE TITLE
Unknown resource type "KMS::Key" in resource deployment map

### ISSUE THREAD

**Issue by peavers** [localstack/localstack#3362]
# Type of request: This is a ...

[X] bug report
[ ] feature request

# Detailed description
Using the latest docker image, and aws-cdk 1.78.0 (java) 

Attempting to deploy a template which depends heavily on KMS, I am hitting a roadblock with the following error: 
```
:WARNING:localstack.utils.cloudformation.template_deployer: Unknown resource type "KMS::Key" in resource deployment map:
```

The block it apparently has errors with: 
```
{
  'Type': 'AWS::KMS::Key',
  'Properties': {
    'KeyPolicy': {
      'Statement': [
        {
          'Action': [
            'kms:GenerateDataKey',
            'kms:Decrypt'
          ],
          'Effect': 'Allow',
          'Principal': {
            'Service': 'sns.amazonaws.com'
          },
          'Resource': '*'
        },
        {
          'Action': [
            'kms:GenerateDataKey',
            'kms:Decrypt'
          ],
          'Effect': 'Allow',
          'Principal': {
            'AWS': {
              'Fn::Join': [
                '',
                [
                  'arn:',
                  {
                    'Ref': 'AWS::Partition'
                  },
                  ':iam::',
                  {
                    'Ref': 'AWS::AccountId'
                  },
                  ':root'
                ]
              ]
            }
          },
          'Resource': '*'
        },
        {
          'Action': [
            'kms:Create*',
            'kms:Describe*',
            'kms:Enable*',
            'kms:List*',
            'kms:Put*',
            'kms:Update*',
            'kms:Revoke*',
            'kms:Disable*',
            'kms:Get*',
            'kms:Delete*',
            'kms:ScheduleKeyDeletion',
            'kms:CancelKeyDeletion',
            'kms:GenerateDataKey',
            'kms:TagResource',
            'kms:UntagResource'
          ],
          'Effect': 'Allow',
          'Principal': {
            'AWS': {
              'Fn::Join': [
                '',
                [
                  'arn:',
                  {
                    'Ref': 'AWS::Partition'
                  },
                  ':iam::',
                  {
                    'Ref': 'AWS::AccountId'
                  },
                  ':root'
                ]
              ]
            }
          },
          'Resource': '*'
        }
      ],
      'Version': '2012-10-17'
    },
    'EnableKeyRotation': True,
  },
  'UpdateReplacePolicy': 'Retain',
  'DeletionPolicy': 'Retain'
}
```

## Expected behavior
KMS is supported and the resource type is found. 

## Actual behavior
Looks like the resource type isn't found...

**Comment by peavers** [comment]
I have an example of this as a standalone repo https://github.com/peavers/kms-error-example if it helps at all

**Comment by whummer** [comment]
Thanks for reporting @peavers . This should be fixed in #3459 - can you please pull the `latest` Docker image and give it another try? Please report here if the problem persists. Thanks
